source/includes/v2/_odsql.md

```diff
@@ -134,17 +134,66 @@ Supported geometry expressions are:
 
 A scalar function can be used in [select arithmetic expressions](#arithmetic-select-expression), [filter expressions](#filter-expression) or in the [group by clause](#group-by-clause).
 
-Function|Parameters|Description|Limitation
---------|-----------|----------|----------
-`length`|string literal or string field literal|Returns the number of characters|
-`now`|no parameter|Returns the current date|Only works on filter expressions
-`year`|date field literal|Returns the year of a date|
-`month`|date field literal|Returns the month of a date|
-`day`|date field literal|Returns the day of a date|
-`hour`|date field literal|Returns the hour of a date|
-`minute`|date field literal|Returns the minute of a date|
-`second`|date field literal|Returns the second of a date|
-`date_format`|date field literal|Returns a formatted date|
+Function|Parameters|Description
+--------|-----------|----------
+`length`|string literal or string field literal|Returns the number of characters
+`now`|optional named [parameters](#parameters-to-the-now-function)|Returns the current date
+`year`|date field literal|Returns the year of a date
+`month`|date field literal|Returns the month of a date
+`day`|date field literal|Returns the day of a date
+`hour`|date field literal|Returns the hour of a date
+`minute`|date field literal|Returns the minute of a date
+`second`|date field literal|Returns the second of a date
+`date_format`|date field literal|Returns a formatted date
+
+#### Parameters to the now() function
+
+> Examples, assuming the current date time is 2021-05-06 12:34:55.450500+00:00, which is a Thursday
+
+```sql
+now() -- Returns '2021-05-06T12:34:55.450500+00:00'
+now(year=2000) -- Sets the year component to return '2000-05-06T12:34:55.450500+00:00'
```

Comment:
I know it's well documented with extensive parameters documentation below but it may be misleading to not have an example with relative year here, and it's used a lot(?)
```suggestion
now(year=2000) -- Sets the year component to return '2000-05-06T12:34:55.450500+00:00'
now(years=-1) -- Sets the year to one year ago  which is '2020-05-06T12:34:55.450500+00:00'
```

Reply:
Thanks. There is just one extra space in your suggestion (or a missing comma maybe)

Reply:
fixed!